addons/mail/models/mail_message.py

```diff
@@ -898,6 +898,7 @@ def _message_format(self, fnames, format_reply=True):
                 'tracking_value_ids': tracking_value_ids,
                 'messageReactionGroups': reaction_groups,
                 'record_name': record_name,
+                'internal': message_sudo.subtype_id.internal if message_sudo.subtype_id else False
```

Comment:
Why do you need it ? And if really necessary, would name it ``subtype_internal`` to avoid mixin with "is_internal" flag of message.

Reply:
`is_internal` is used to handle non-"log note" type messages and allow an administrator to toggle the ability for portal users to view a message with an enabled boolean_toggle.
`subtype_internal` is used to display "log note" type messages in the portal chatter to administrators.
My intention was to show that those messages can never be viewed by portal users with a disabled boolean_toggle.